subs/pantry/src/Pantry/Hackage.hs

```diff
@@ -178,11 +200,13 @@ updateHackageIndex mreason = do
               if oldHash == oldHashCheck
                 then oldSize <$ logInfo "Updating preexisting cache, should be quick"
                 else 0 <$ do
-                  logInfo "Package index change detected, that's pretty unusual"
-                  logInfo $ "Old size: " <> display oldSize
-                  logInfo $ "Old hash (orig) : " <> display oldHash
-                  logInfo $ "New hash (check): " <> display oldHashCheck
-                  logInfo "Forcing a recache"
+                  logWarn $ mconcat [
```

Comment:
Not a big deal, but `mconcat` has more overhead than using a bunch of `<>`s, since it forces an intermediate list representation.

Reply:
Just because of this :
> that's pretty unusual
it doesn't seem like performance here is the main concern here :) Moreover I really hope hackage folks are not gonna start messing with the index file and cause this warning to appear.

Using `mconcat` here is more readable in my opinion. But as always, if you'd like me to change it to bunch of `<>`, I'll do that.

Reply:
Nah, not worth changing, it's a common comment I add on PRs just to make the issue known.